### PR TITLE
"We didn’t realise we were making memories, we just knew we were having fun." - Winnie the Pooh

### PR DESCRIPTION
**Pull request by PseudoDistant**
I need to fix my IDEA setup...

### CODE REVIEW

**Comment by PseudoDistant** [comment]
Accidentally pushed to the org instead of my branch.

**Review by thecatcore — APPROVED** [review]

**Review by PseudoDistant — COMMENTED** [review]
Yes

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Legacy Rewoven API is the library for essential hooks and interoperability mecha
 
 
 
-For support, consider joining the [Legacy Rewoven discord server](https://discord.gg/K9U4PfmFfp), opening an issue, or creating a discussion.
+For support, consider joining the [Legacy Fabric discord server](https://discord.gg/emgF7bp), opening an issue, or creating a discussion.
```
Comment (thecatcore) [inline README.md]:
```suggestion
For support, consider joining the [Legacy Fabric discord server](https://legacyfabric.net/discord), opening an issue, or creating a discussion.
```

File: build.gradle [hunk 2]
```diff
@@ -294,7 +291,7 @@ dependencies {
 	afterEvaluate {
 		subprojects.each {
 			if (it.name != project.testmods_project) {
-				implementation project(path: ":${it.name}", configuration: "dev")
+//				implementation project(path: ":${it.name}", configuration: "dev")
```
Comment (thecatcore) [inline build.gradle]:
```suggestion
				implementation project(path: ":${it.name}", configuration: "dev")
```

File: modules/base/src/main/resources/fabric.mod.json [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/base/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/client/keybindings/src/main/resources/fabric.mod.json [hunk 4]
```diff
@@ -1,26 +1,25 @@
...
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/client/keybindings/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/client/rendering/src/main/resources/fabric.mod.json [hunk 5]
```diff
@@ -1,22 +1,22 @@
...
   "version": "${version}",
   "environment": "client",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/client/rendering/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/crash-report-info/src/main/resources/fabric.mod.json [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/crash-report-info/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/entity-events/src/main/resources/fabric.mod.json [hunk 7]
```diff
@@ -1,24 +1,22 @@
...
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/entity-events/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/lifecycle-events/src/main/resources/fabric.mod.json [hunk 8]
```diff
@@ -1,24 +1,23 @@
...
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/lifecycle-events/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/networking/src/main/resources/fabric.mod.json [hunk 9]
```diff
@@ -1,23 +1,22 @@
...
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/networking/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/resources/src/main/resources/fabric.mod.json [hunk 10]
```diff
@@ -1,18 +1,17 @@
...
+  "name": "Legacy Fabric Resource Loader (V1)",
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
   "icon": "assets/legacy-fabric/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/resources/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/server/gamerules/src/main/resources/fabric.mod.json [hunk 11]
```diff
@@ -1,18 +1,17 @@
...
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/server/gamerules/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/server/permissions/src/main/resources/fabric.mod.json [hunk 12]
```diff
@@ -1,19 +1,18 @@
...
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline modules/server/permissions/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/server/sponge-commands/src/main/resources/fabric.mod.json [hunk 13]
```diff
@@ -1,11 +1,11 @@
...
-  "name": "Legacy Rewoven Sponge Command API (v2)",
+  "id": "legacy-fabric-sponge-command-api-v2",
+  "name": "Legacy Fabric Sponge Command API (v2)",
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric/icon.png",
   "contact": {
     "homepage": "https://legacyrewoven.github.io/",
```
Comment (thecatcore) [inline modules/server/sponge-commands/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/server/vanilla-commands/src/main/resources/fabric.mod.json [hunk 14]
```diff
@@ -1,24 +1,23 @@
...
-  "name": "Legacy Rewoven Vanilla Command API (V0)",
+  "id": "legacy-fabric-vanilla-command-api-v1",
+  "name": "Legacy Fabric Vanilla Command API (V0)",
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric/icon.png",
   "contact": {
     "homepage": "https://legacyrewoven.github.io/",
```
Comment (thecatcore) [inline modules/server/vanilla-commands/src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/server/vanilla-commands/src/main/resources/fabric.mod.json [hunk 15]
```diff
@@ -1,24 +1,23 @@
...
     "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
```
Comment (thecatcore) [inline modules/server/vanilla-commands/src/main/resources/fabric.mod.json]:
```suggestion
    "issues": "https://github.com/Legacy-Fabric/fabric/issues",
```

File: modules/server/vanilla-commands/src/main/resources/fabric.mod.json [hunk 16]
```diff
@@ -1,24 +1,23 @@
...
     "sources": "https://github.com/LegacyRewoven/rewoven-api"
```
Comment (thecatcore) [inline modules/server/vanilla-commands/src/main/resources/fabric.mod.json]:
```suggestion
    "sources": "https://github.com/Legacy-Fabric/fabric"
```

File: src/main/resources/fabric.mod.json [hunk 17]
```diff
@@ -1,21 +1,35 @@
...
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric-api/icon.png",
   "contact": {
-    "homepage": "https://legacyrewoven.github.io/",
-    "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
-    "sources": "https://github.com/LegacyRewoven/rewoven-api"
+    "homepage": "https://legacy-fabric.github.io/",
```
Comment (thecatcore) [inline src/main/resources/fabric.mod.json]:
```suggestion
    "homepage": "https://legacyfabric.net/",
```

File: modules/server/vanilla-commands/src/main/resources/fabric.mod.json [hunk 18]
```diff
@@ -1,24 +1,23 @@
...
   "schemaVersion": 1,
-  "id": "legacy-rewoven-vanilla-command-api-v1",
-  "name": "Legacy Rewoven Vanilla Command API (V0)",
+  "id": "legacy-fabric-vanilla-command-api-v1",
+  "name": "Legacy Fabric Vanilla Command API (V0)",
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric/icon.png",
```
Comment (thecatcore) [inline modules/server/vanilla-commands/src/main/resources/fabric.mod.json]:
```suggestion
  "icon": "assets/legacy-fabric-api/icon.png",
```

File: modules/server/sponge-commands/src/main/resources/fabric.mod.json [hunk 19]
```diff
@@ -1,11 +1,11 @@
...
   "schemaVersion": 1,
-  "id": "legacy-rewoven-sponge-command-api-v2",
-  "name": "Legacy Rewoven Sponge Command API (v2)",
+  "id": "legacy-fabric-sponge-command-api-v2",
+  "name": "Legacy Fabric Sponge Command API (v2)",
   "version": "${version}",
   "environment": "*",
   "license": "Apache-2.0",
-  "icon": "assets/legacy-rewoven-api/icon.png",
+  "icon": "assets/legacy-fabric/icon.png",
```
Comment (thecatcore) [inline modules/server/sponge-commands/src/main/resources/fabric.mod.json]:
```suggestion
  "icon": "assets/legacy-fabric-api/icon.png",
```

File: modules/server/sponge-commands/src/main/resources/fabric.mod.json [hunk 20]
```diff
@@ -1,11 +1,11 @@
...
   "contact": {
     "homepage": "https://legacyrewoven.github.io/",
     "issues": "https://github.com/LegacyRewoven/rewoven-api/issues",
```
Comment (thecatcore) [inline modules/server/sponge-commands/src/main/resources/fabric.mod.json]:
```suggestion
    "issues": "https://github.com/Legacy-Fabric/fabric/issues",
```

**Review by thecatcore — CHANGES_REQUESTED** [review]

**Comment by thecatcore** [comment]
You also have to fix style regex

**Comment by PseudoDistant** [comment]
I still had to fix checkstyle...

**Pull request merged**